tests/test_database.py

```diff
@@ -83,19 +83,16 @@ def test_add_workout(self):
 
         database.add_workout(new_workout)
         # Getting the new user by their username
-        db_user = database.get_user_class_by_username(new_user.username)
+
         # Getting the workout by their name
         db_workout = database.get_workout_class_by_name(new_workout.name)
 
-        # Setting our current workout object's id as mongodb id of user in string format
-        new_workout.creator_id = str(db_user.id)
-
         # Sets the id
         new_workout.id = db_workout.id
 
         #sets creator's id
-        new_workout.creator_id = str(db_user.id)
-        self.assertTrue(str(db_user.id) == new_workout.creator_id)
+        new_workout.creator_id = db_user.id
+        self.assertTrue(db_user.id == new_workout.creator_id)
 
```

Comment:
Add an assert that compares the db_workout.as_dict() with new_workout.as_dict() to see if they are the same. 

`self.assertTrue(db_workout.as_dict() == new_workout.as_dict())`